### PR TITLE
Update Developer's Guide.md

### PR DESCRIPTION
**Pull request by Centerville1**
Dr. Tuck recommended I add this, after I ran into an issue related to not restarting everything.

### CODE REVIEW

**Review by bennettforkner — APPROVED** [review]

**Comment by Centerville1** [comment]
Just a heads up @bennettforkner, I don't have permission to merge this, so one of the admins will need to

**Comment by bennettforkner** [comment]
> Just a heads up @bennettforkner, I don't have permission to merge this, so one of the admins will need to

Even with approval? Strange. I would have thought you would have general write access to the repository. Up to @russtuck

**Comment by Centerville1** [comment]
> > Just a heads up @bennettforkner, I don't have permission to merge this, so one of the admins will need to
> 
> Even with approval? Strange. I would have thought you would have general write access to the repository. Up to @russtuck

Yeah, it said something about a protected branch, which is strange, but maybe the documentation has some extra security in place.

**Pull request merged**
